components/filefeed/setup-space.tsx

```diff
@@ -41,6 +41,8 @@ export const SetupSpace = () => {
           onSubmit={handleSubmit}
           className="mb-8"
         >
+          <input type="hidden" name="focusBgColor" value="#616A7D" />
+          <input type="hidden" name="backgroundColor" value="#090B2B" />
```

Comment:
Where do these get pulled in?

Reply:
![CleanShot 2023-05-02 at 12 03 07](https://user-images.githubusercontent.com/12041898/235721725-0775f8b6-d33e-422b-ae71-af70548d51a6.png)
![CleanShot 2023-05-02 at 12 03 49](https://user-images.githubusercontent.com/12041898/235721776-046cf201-0e55-4b7c-96fd-49eeaa2672e5.png)